### PR TITLE
Update README for CT log 2022 key

### PR DESCRIPTION
**Pull request by Hayden-IO**
Signed-off-by: Hayden B <hblauzvern@google.com>

### CODE REVIEW

File: README.md
Comment (Hayden-IO) [inline README.md]: This path doesn't exist yet, but it will soon enough

**Review by dlorenc — DISMISSED** [review]

**Comment by asraa** [comment]
Will merge after we stage the new root

**Comment by Hayden-IO** [comment]
This should be good to merge, just need approval

**Review by cpanato — APPROVED** [review]

**Review by asraa — APPROVED** [review]
Thank you!

**Pull request merged**
